src/js/components/Image.js

```diff
@@ -0,0 +1,69 @@
+import classNames from 'classnames/dedupe';
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import Util from '../utils/Util';
+
+const METHODS_TO_BIND = ['onImageError'];
+
+class Image extends React.Component {
+  constructor() {
+    super(...arguments);
+
+    this.state = {
+      imageErrorCount: 0
+    };
+
+    METHODS_TO_BIND.forEach((method) => {
+      this[method] = this[method].bind(this);
+    });
+  }
+
+  componentWillReceiveProps(nextProps) {
+    // Try again if new urls were provided
```

Comment:
Arh, right! I always forget that that is the case. Good catch. I updated the PR to account for that